File: async-scala-client/src/main/scala/io/swagger/client/model/TransactionForSefazGoods.scala
package io.swagger.client.model

import org.joda.time.DateTime
import java.util.UUID


case class TransactionForSefazGoods (
  header: Option[HeaderForGoods],
lines: Option[List[LineForSefazGoods]],
calculatedTaxSummary: Option[CalculatedTaxSummaryForGoods]
)

File: javascript-closure-angular-client/API/Client/TransactionForSefazGoods.js
goog.provide('API.Client.TransactionForSefazGoods');

/**
 * @record
 */
API.Client.TransactionForSefazGoods = function() {}

/**
 * @type {!API.Client.HeaderForGoods}
 * @export
 */
API.Client.TransactionForSefazGoods.prototype.header;

/**
 * @type {!Array<!API.Client.LineForSefazGoods>}
 * @export
 */
API.Client.TransactionForSefazGoods.prototype.lines;

/**
 * @type {!API.Client.CalculatedTaxSummaryForGoods}
 * @export
 */
API.Client.TransactionForSefazGoods.prototype.calculatedTaxSummary;

